### PR TITLE
Tutorial emily

### PR DESCRIPTION
**Pull request by emilyyyzh**
- Tutorial level scene currently named 'tutorial-emily' but can be renamed later. Probably best to play it first before looking at the code. I won't talk about the goal or how the tutorial plays so you can better judge how it is.
- Added a special object that the user needs to pick up to beat the tutorial. I haven't had the time to look into scriptable objects yet but I'm thinking we make this special object a scriptable object that when you pick up, you beat the tutorial and something happens (like a scene shift). Would that be a possibility?

### CODE REVIEW

File: Assets/Scripts/UI/TutorialPlayerThoughts.cs [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+            else if (_num_bounces > 3)
+            {
+                playerThoughts.text = "Maybe the control panel will be helpful.";
+            }
+            else
+            {
+                playerThoughts.text = "Ouch. The water is electrified by the power generator.";
+            }
+
+        }
```
Comment (wang-henry) [inline Assets/Scripts/UI/TutorialPlayerThoughts.cs:31]: I think we can move this logic to under the `OnControllerColliderHit`. That way we would not be doing these checks every frame. So something like this maybe?

```C#
if (hit.gameObject.CompareTag("puddle"))
{
    _num_bounces += 1;
    
    if (_num_bounces > 6)
    { .. }
    etc...
}
  ```

Comment (emilyyyzh) [reply Assets/Scripts/UI/TutorialPlayerThoughts.cs]: fixed, thanks for the catch :D

**Review by wang-henry — APPROVED** [review]
Code looks good to me.

**Review by ectodrop — APPROVED** [review]
looks good, I'll polish up a few things before the alpha demo. nicely done! :+1:

**Pull request merged**